src/librustc_typeck/check/dropck.rs

```diff
@@ -238,14 +238,12 @@ fn ensure_drop_predicates_are_implied_by_item_defn<'tcx>(
 /// Let `v` be some value (either temporary or named) and 'a be some
 /// lifetime (scope). If the type of `v` owns data of type `D`, where
 ///
-///   (1.) `D` has a lifetime- or type-parametric Drop implementation, and
-///   (2.) the structure of `D` can reach a reference of type `&'a _`, and
-///   (3.) either:
-///
-///     (A.) the Drop impl for `D` instantiates `D` at 'a directly,
+/// * (1.) `D` has a lifetime- or type-parametric Drop implementation, and
```

Comment:
I wanted to use the capital letters A and B explicitly for the sub-bullets (which as far as I know is not supported by markdown).  So that motivated me to use asterisks for the sub-bullets; then I just used asterisks throughout for consistency.